recipes/bertha/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "bertha" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/cph-tum/{{ name }}/-/archive/master/{{ name }}-master.tar.gz
```

Comment:
Is there a tag 1.0.0 upstream?

Reply:
Not yet, but as soon as there is I shall update this line.